parser/preprocessed/SqlLexer_gen.cpp

```diff
@@ -8,8 +8,8 @@
 
 #define FLEX_SCANNER
 #define YY_FLEX_MAJOR_VERSION 2
-#define YY_FLEX_MINOR_VERSION 6
-#define YY_FLEX_SUBMINOR_VERSION 0
+#define YY_FLEX_MINOR_VERSION 5
```

Comment:
Do I need to build the code on a machine with FLEX version as 2.6.0 and check the preprocessed files in?